client/src/store/features/users/usersSlice.js

```diff
@@ -46,6 +23,8 @@ export const usersSlice = createSlice({
       ...state,
       id: action.payload.id,
       username: action.payload.username,
+      email: action.payload.username,
```

Comment:
Shouldn't this be `email: action.payload.email`?

Reply:
Correct, it was a typo.